extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -94,8 +98,7 @@ public void run() {
                             lastBridgeConnectionState = true;
                             onConnectionResumed(bridge);
                         }
-                    }
-                    if (lastBridgeConnectionState) {
```

Comment:
The lastBridgeConnectionState potentially gets changed in the above block. As a result, the below block was executed. With your change, it won't anymore.